packages/subgraph/src/mappingHelpers.ts

```diff
@@ -154,6 +198,34 @@ export function getOrInitSuperToken(
     return token as Token;
 }
 
+export function getOrInitDefaultGovernanceConfigs(
+    block: ethereum.Block
+): DefaultGovernanceConfigs {
+    let defaultGovernanceConfigs = DefaultGovernanceConfigs.load(
+        ZERO_ADDRESS.toHexString()
+    );
+
+    if (defaultGovernanceConfigs == null) {
+        defaultGovernanceConfigs = new DefaultGovernanceConfigs(
+            ZERO_ADDRESS.toHexString()
+        );
+        defaultGovernanceConfigs.createdAtTimestamp = block.timestamp;
+        defaultGovernanceConfigs.createdAtBlockNumber = block.number;
+        defaultGovernanceConfigs.rewardAddress = ZERO_ADDRESS;
+        defaultGovernanceConfigs.liquidationPeriod = BIG_INT_ZERO;
```

Comment:
because it is the true "default", default in this case refers to what is set for a config (e.g. rewardAddress, liqPeriod, patPeriod, minDeposit) where the superToken field is `address(0)`.
each of these fields are only set at the protocol (and therefore the subgraph) when an event is emitted indicating that the default value has been set for a particular config.
it is better to initialize the entity with properties = 0 which means nothing has been set yet than to initialize at a value (which may be incorrect) we've used in the past.